readme.md

```diff
@@ -0,0 +1,369 @@
+# LazyLoad Plugin – Lazy Load Images, Videos, and Iframes
+
+The best free lazy load plugin for WordPress. Lazy load images, videos, and iframes to improve performance and Core Web Vitals scores.
```

Comment:
Add a link to the plugin page on wordpress.org https://wordpress.org/plugins/rocket-lazy-load/?
This readme (md version) is only for GitHub right, not used in the plugin information so I think a link might a be good idea.

```suggestion
The best free lazy load plugin for WordPress. Lazy load images, videos, and iframes to improve performance and Core Web Vitals scores.
[Download it on wordpress.org](https://wordpress.org/plugins/rocket-lazy-load/).
```